platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/UiSettings.java

```diff
@@ -86,6 +86,19 @@ public void setCompassGravity(int gravity) {
     }
 
     /**
+     * Enables or disables fading of the compass when facing north.
+     * <p>
+     * By default this feature is enabled
+     * </p>
+     *
+     * @param compassFadeFacingNorth True to enable the fading animation; false to disable it
+     */
+    public void setCompassFadeFacingNorth(boolean compassFadeFacingNorth) {
+        compassSettings.setFadeFacingNorth(compassFadeFacingNorth);
+        mapView.setCompassFadeFacingNorth(compassFadeFacingNorth);
+    }
```

Comment:
When we do `mapView.setCompassFadeFacingNorth(compassFadeFacingNorth)` we're already doing `compassSettings.setFadeFacingNorth(compassFadeFacingNorth);`, could we remove `compassSettings.setFadeFacingNorth(compassFadeFacingNorth);` from here?